gnd/src/main/java/com/google/android/gnd/model/feature/FeatureMutation.java

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-package com.google.android.gnd.persistence.shared;
+package com.google.android.gnd.model.feature;
 
 import static com.google.android.gnd.util.ImmutableListCollector.toImmutableList;
 import static java8.util.stream.StreamSupport.stream;
 
-import com.google.android.gnd.vo.Point;
+import com.google.android.gnd.model.Mutation;
```

Comment:
Correct! Tried to split the classes up into coarse-grained functional categories rather than horizontal ones (mutations, config, data, etc) since I think they're a bit easier to grok. We're not using packages to enforce member visibility in this case, so prioritizing readability.